app/presenters/menu/settings_loader.rb

```diff
@@ -0,0 +1,36 @@
+module Menu
+  class SettingsLoader < YamlLoader
+    include Singleton
+
+    def self.load
+      instance.load_from_settings
```

Comment:
As this is effectively a cache, what happens when someone changes the settings?  Are we expecting it to show the new menus or do we expect them to bounce the workers?

Reply:
Yes, it's a cache and I planned to document that a worker restart is needed for this to take effect. Do you have a better idea?